tools/generate_emails.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python3
+import os
+
+from premailer import Premailer
+from cssutils import profile
+from cssutils.profiles import Profiles, properties, macros
+
+if __name__ == "__main__":
+    mapping = (('%7B%7B%20', '{{ '), ('%20%7D%7D', ' }}'))
```

Comment:
let's call this variable: escaped_jinja2_characters, and make it a list of tuples.